cider/tests/utils/CiderTestBase.h

```diff
@@ -65,6 +73,8 @@ class CiderTestBase : public testing::Test {
   std::string table_name_;
   std::string create_ddl_;
   std::vector<std::shared_ptr<CiderBatch>> input_;
+  ArrowArray* array = nullptr;
```

Comment:
protected variable's name should been end with _

Reply:
Get it, thanks~